src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/NewAzureVpnClientConfigurationCommand.cs

```diff
@@ -0,0 +1,152 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using AutoMapper;
+using Microsoft.Azure.Commands.Network.Models;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Management.Automation;
+using System.Security.Cryptography.X509Certificates;
+using MNM = Microsoft.Azure.Management.Network.Models;
+
+namespace Microsoft.Azure.Commands.Network
+{
+    [Cmdlet(VerbsCommon.New, "AzureRmVpnClientConfiguration"), OutputType(typeof(string))]
```

Comment:
Yes - per update from Aanand in the mail, we do not want a StorageContext here. To quote from the mail - "We already have a cmdlet that outputs a URL string. Its been there for about 3 to 4 years now and customers have been perfectly fine using it. There have been no CRIs or other feedback to change the experience. The new cmdlet below creates profiles for some additional scenarios. Since customers are already habituated to using the older cmdlet we want to provide the same experience for the new one also"